include/loads_bcs/absorbing_constraint.h

```diff
@@ -15,9 +15,17 @@ class AbsorbingConstraint {
   //! \param[in] h_min Cell height
   //! \param[in] a Equation constant
   //! \param[in] b Equation constant
+  //! \param[in] position Nodal position along boundary
   AbsorbingConstraint(int setid, unsigned dir, double delta, double h_min,
-                      double a = 1, double b = 1)
-      : setid_{setid}, dir_{dir}, delta_{delta}, h_min_{h_min}, a_{a}, b_{b} {};
+                      double a = 1., double b = 1.,
+                      std::string position = "null")
```

Comment:
@jgiven100 could we enum the position labels instead of string? C++ class enums would be the best. Thanks!

Reply:
@kks32 is adding the enum class to data_types.h ok?

Reply:
Yes that would be a good place to add enum class

Reply:
Resolved